### PR TITLE
adds Explore option for explorable datasets

### PR DESCRIPTION
**Pull request by astewes**
-  Adds Explore option to Jump menu (Tracker home + detail) for explorable datasets

explorable

### CODE REVIEW

File: cdap-ui/app/features/tracker/controllers/results-ctrl.js [hunk 1]
```diff
@@ -150,6 +150,9 @@ class TrackerResultsController {
         createDate: entity.metadata.SYSTEM.properties['creation-time'],
         datasetType: entity.metadata.SYSTEM.properties.type
       });
+      if(entity.metadata.SYSTEM.tags.indexOf('explore') !== -1) {
```
Comment (anwar6953) [inline cdap-ui/app/features/tracker/controllers/results-ctrl.js:153]: I think we should use the same logic here, that we use to determine whether we want to show the 'explore' tab here (screenshot below). @ajainarayanan thoughts?

image

Comment (ajainarayanan) [inline cdap-ui/app/features/tracker/controllers/results-ctrl.js:153]: @anwar6953 Right now the logic which we use to find if a dataset is explorable or not is wrong in CDAP. we should use this method in CDAP to find if its explorable. Its the otherway :)

**Comment by ajainarayanan** [comment]
👍 This might not be entirely true today I believe (asserting a dataset is explorable based on system tag) but in release/3.5 backend will be fixed to make sure we have this as the only source of truth for finding whether any dataset is explorable or not.

**Comment by elfenheart** [comment]
Add explore link to streams. Streams are always explorable

**Comment by elfenheart** [comment]
👍  LGTM

File: cdap-ui/app/features/tracker/controllers/results-ctrl.js (hunk 1 above)
Comment (anwar6953) [inline cdap-ui/app/features/tracker/controllers/results-ctrl.js:153]: Do you recall how we determine to show explore tab on the dataset page?

Comment (anwar6953) [inline cdap-ui/app/features/tracker/controllers/results-ctrl.js:153]: Just suggesting that the UI should be consistent in how it determines whether explore is enabled or not.

**Pull request merged**